src/Storage/Storage.Management/help/New-AzStorageContext.md

```diff
@@ -61,6 +61,18 @@ New-AzStorageContext [-StorageAccountName] <String> [-UseConnectedAccount] [-Pro
  -Environment <String> [<CommonParameters>]
 ```
 
+### AccountNameAndKeyServiceEndpoint
+```
+New-AzStorageContext [-StorageAccountName] <String> [-StorageAccountKey] <String> -BlobEndpoint <String>
```

Comment:
I'm thinking whether StorageAccountKey should be wrapped with securestring.

Reply:
We just reuse the original -StorageAccountKey parameter in New-AzStorageContext.
Since there are many use already use this, we don't want to change the behavior here, to give user an aligned usage for this cmdlet.